contracts/tasks/runner/prove.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-console */
+import { task } from "hardhat/config";
```

Comment:
maybe let's remove wip file until the pr where you implement it?

Reply:
Let's keep it, I'll add this task in next PR.